cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -80,6 +81,13 @@ type CMServer struct {
 	ServiceAccountKeyFile   string
 	RootCAFile              string
 
+	// volumeConfig
+	PersistentVolumeRecyclerDefaultScrubPod          string
+	PersistentVolumeRecyclerMinTimeoutNfs            int
```

Comment:
Do you expect this to be the norm or the exception?

Do you think that a single config struct would be sufficient for all volumes?  That sounds much more attractive - a map[string]config struct